app/assets/javascripts/users.js.erb

```diff
@@ -50,6 +40,37 @@ function initMap() {
   }
 }
 
+function displayTargets() {
+  $.get('/user_targets').done(function(data) {
```

Comment:
You should add the path as a hidden value or data attribute on the html. Then you get the path from it and use it here. Doing this makes it easier to change the path if you use several times.